packages/lending/src/protocols/aave-v3/configs.ts

```diff
@@ -171,6 +180,57 @@ export const configs: Config[] = [
       },
     ],
   },
+  // https://github.com/bgd-labs/aave-address-book/blob/main/src/AaveV3Gnosis.sol
+  {
+    chainId: common.ChainId.gnosis,
+    contractMap: {
+      Pool: '0xb50201558B00496A145fE76f7424749556E326D8',
+      PoolDataProvider: '0x501B4c19dd9C2e06E94dA7b6D5Ed4ddA013EC741',
+      AaveOracle: '0xeb0a051be10228213BAEb449db63719d6742F7c4',
+    },
+    reserves: [
+      {
+        asset: gnosisTokens.WETH,
+        aToken: gnosisTokens.aGnoWETH,
+        used: { deposit: true, withdraw: true, borrow: true, repay: true, flashLoan: true },
+      },
+      {
+        asset: gnosisTokens.wstETH,
+        aToken: gnosisTokens.aGnowstETH,
+        used: { deposit: true, withdraw: true, borrow: true, repay: true, flashLoan: true },
+      },
+      {
+        asset: gnosisTokens.GNO,
+        aToken: gnosisTokens.aGnoGNO,
+        used: { deposit: true, withdraw: true, borrow: true, repay: true, flashLoan: true },
+      },
+      {
+        asset: gnosisTokens.USDC,
+        aToken: gnosisTokens.aGnoUSDC,
+        used: { deposit: true, withdraw: true, borrow: true, repay: true, flashLoan: true },
+      },
+      {
+        asset: gnosisTokens.xDAI,
+        aToken: gnosisTokens.aGnoWXDAI,
+        used: { deposit: true, withdraw: true, borrow: true, repay: true, flashLoan: true },
```

Comment:
native token can not flash loan
can ref [line 95](https://github.com/dinngo/protocolink-js-sdk/blob/master/packages/lending/src/protocols/aave-v3/configs.ts#L95)